src/wallet/sqlite.cpp

```diff
@@ -119,6 +121,28 @@ bool SQLiteDatabase::Verify(bilingual_str& error)
 {
     assert(m_db);
 
+    // Check the application ID matches our network magic
+    sqlite3_stmt* app_id_stmt{nullptr};
+    int ret = sqlite3_prepare_v2(m_db, "PRAGMA application_id", -1, &app_id_stmt, nullptr);
+    if (ret != SQLITE_OK) {
+        sqlite3_finalize(app_id_stmt);
+        error = strprintf(_("SQLiteDatabase: Failed to prepare the statement to fetch the application id: %s"), sqlite3_errstr(ret));
+        return false;
+    }
+    ret = sqlite3_step(app_id_stmt);
+    if (ret != SQLITE_ROW) {
+        sqlite3_finalize(app_id_stmt);
+        error = strprintf(_("SQLiteDatabase: Failed to fetch the application id: %s"), sqlite3_errstr(ret));
+        return false;
+    }
+    uint32_t app_id = (uint32_t)sqlite3_column_int(app_id_stmt, 0);
```

Comment:
87e2d29baca9d08e90ed401bf360b9b7e866c62c
```suggestion
    uint32_t app_id = static_cast<uint32_t>(sqlite3_column_int(app_id_stmt, 0));
```

Reply:
Done